src/Elliptic/DiscontinuousGalerkin/DgElementArray.hpp

```diff
@@ -0,0 +1,126 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "AlgorithmArray.hpp"
+#include "Domain/ElementIndex.hpp"
+#include "Domain/InitialElementIds.hpp"
+#include "Domain/Tags.hpp"
+#include "Elliptic/DiscontinuousGalerkin/InitializeElement.hpp"
+#include "IO/Observer/TypeOfObservation.hpp"
+#include "Parallel/ConstGlobalCache.hpp"
+#include "Parallel/Info.hpp"
+#include "Parallel/Invoke.hpp"
+#include "Parallel/ParallelComponentHelpers.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace observers {
+namespace Actions {
+template <observers::TypeOfObservation TypeOfObservation>
+struct RegisterWithObservers;
+}  // namespace Actions
+}  // namespace observers
+/// \endcond
+
+namespace Elliptic {
+
+template <class Metavariables, class ActionList>
+struct DgElementArray {
```

Comment:
Add comment that this is the same as the evolution code right now, but may diverge in the future (e.g. for multigrid)